### ISSUE TITLE
Alternative for creating docker instances within cloud

### ISSUE THREAD

**Issue by basplijnaer** [AmpersandTarski/RAP#173]

**Comment by Michiel-s** [comment]
Indeed. This is a complex task. Will also require refactoring backend code that is responsible for spinning up containers for student prototypes.

**Comment by Michiel-s** [comment]
That is done in this files: RAP4/customizations/bootstrap/files/ExecEngineFunctions.php line 292
